### ISSUE TITLE
NVDA sometimes reads out wrong data

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/resistance-in-a-wire#202]
**Test device:**
Dell
**Operating System:**
Win 10
**Browser:**
firefox
**Problem description:**
For https://github.com/phetsims/QA/issues/239
At times, when reading through the values the screen reader will read out the wrong values for the sliders.  This was recorded at the end of the scene description.  It seems to occur a bit more if manipulation occurred with a mouse rather than keyboard nav, but I could be wrong.  I've been unable to reliably reproduce.
**Steps to reproduce:**
1. Use firefox and nvda.
2. Manipulate sliders
3. Listen to scene description and listen for discrepancies 

**Screenshots:**
https://drive.google.com/file/d/1ileL9npBkrJccjMWFUndONIOYZBlJwfs/view?usp=sharing
https://drive.google.com/file/d/1fUvq8rdlxe1gBL4fpAVBW9J8FCF_yNhE/view?usp=sharing

Troubleshooting information (do not edit):

<details>

Name: ‪Resistance in a Wire‬
URL: https://phet-dev.colorado.edu/html/resistance-in-a-wire/1.6.0-rc.2/phet/resistance-in-a-wire_en_phet.html
Version: 1.6.0-rc.2 2018-12-13 16:36:32 UTC
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:64.0) Gecko/20100101 Firefox/64.0
Language: en-US
Window: 1536x760
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0
Vendor: Mozilla (Mozilla)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {"assert":{"sha":"928741cf","branch":"HEAD"},"axon":{"sha":"de77d4b5","branch":"HEAD"},"brand":{"sha":"1fd6682e","branch":"HEAD"},"chipper":{"sha":"fa2fbadf","branch":"HEAD"},"dot":{"sha":"bbbd8526","branch":"HEAD"},"joist":{"sha":"82521d0c","branch":"HEAD"},"kite":{"sha":"380cef53","branch":"HEAD"},"phet-core":{"sha":"1b90ac2f","branch":"HEAD"},"phet-io":{"sha":"38d7b161","branch":"HEAD"},"phet-io-wrapper-classroom-activity":{"sha":"246085c1","branch":"HEAD"},"phet-io-wrapper-hookes-law-energy":{"sha":"7479b0ec","branch":"HEAD"},"phet-io-wrapper-lab-book":{"sha":"c46f7839","branch":"HEAD"},"phet-io-wrappers":{"sha":"a6bc62ca","branch":"HEAD"},"phetcommon":{"sha":"cd63d89a","branch":"HEAD"},"query-string-machine":{"sha":"06ed6276","branch":"HEAD"},"resistance-in-a-wire":{"sha":"c996a2e3","branch":"HEAD"},"scenery":{"sha":"9953c5f7","branch":"HEAD"},"scenery-phet":{"sha":"15ed6d54","branch":"HEAD"},"sherpa":{"sha":"2cd50500","branch":"HEAD"},"sun":{"sha":"9ee72759","branch":"HEAD"},"tambo":{"sha":"65315b32","branch":"HEAD"},"tandem":{"sha":"ed8f8f1d","branch":"HEAD"}}

</details>

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe. Since NVDA says something like "Length Clickable Slider" before reading values, I think it is actually reading through the sliders and not necessarily the scene description. But I could be wrong I will check both.

**Comment by KatieWoe** [comment]
Might be miscommunication on my part.  I thought that was part of the scene description.  To clarify, it is the part after the equation and wire are read out, and before the menu buttons.  I wasn't able to reproduce to see if it happened earlier in the description.

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe - I have been playing with the sim for about 10 minutes with combinations of mouse and keyboard interaction and haven't been able to see the issue yet. Ill try on my slower laptop.

**Comment by jessegreenberg** [comment]
I also tried the same values that were presented in the video captures.

**Comment by jessegreenberg** [comment]
Hmm, I tried for another 10 minutes or so an a different Windows 10 Devices with FF and NVDA and I haven't been able to see this issue yet. I tried combinations of mouse + keyboard interaction with page up/page down, arrow keys, and shift keys and NVDA is reporting the correct values when I read through the sliders individually with arrow keys  and when I read through the entire sim with "Read All" command insert + down arrow.

**Comment by jessegreenberg** [comment]
I am going to go ahead and close this with wontfix since we haven't been able to observe this again. Thanks for reporting though @KatieWoe and the screen captures were very helpful, we can return to those if this comes up again.

**Comment by KatieWoe** [comment]
Found again during https://github.com/phetsims/QA/issues/257.  Conversation with @jessegreenberg from Slack

>Jesse Greenberg [12:58 PM]
>I believe it is OK to keep closed. How many times did you see it this time? Did it happen consistently?

>Kathryn Woessner [12:59 PM]
>Only saw it once, but we've been doing spot checks, so that might not mean much.
>Also, this time it was in the "first" time that those values are read, rather than the second.  What would you like done?

>Jesse Greenberg [1:03 PM]
>Hmm, that just sounds like maybe a caching issue then.
>(I mean the AT is caching, it does that for other things too.)
>OK, thanks for letting me know. I think OK to proceed without an issue,  I think this is in the noise for AT quirks. Could you please just make a note in the original issue that you saw this again on initial read, but leave closed?

I also believe I was using keyboard nav more than mouse at the time, but still a mix.
